Obsidian/WorldData/BlockUpdates.cs

```diff
@@ -9,171 +9,177 @@ internal static class BlockUpdates
     {
         internal static async Task<bool> HandleFallingBlock(BlockUpdate blockUpdate)
         {
-            if (blockUpdate.block is null) { return false; }
-            var world = blockUpdate.world;
-            var location = blockUpdate.position;
-            var material = blockUpdate.block.Value.Material;
-            if (world.GetBlock(location + Vector.Down) is Block below && (Block.Replaceable.Contains(below.Material) || below.IsFluid))
+            return await Task.Run(() =>
             {
-                world.SetBlock(location, Block.Air);
-                world.SpawnFallingBlock(location, material);
-                return true;
-            }
-            return false;
+                if (blockUpdate.block is null) { return false; }
+                var world = blockUpdate.world;
+                var location = blockUpdate.position;
+                var material = blockUpdate.block.Value.Material;
+                if (world.GetBlock(location + Vector.Down) is Block below && (Block.Replaceable.Contains(below.Material) || below.IsFluid))
+                {
+                    world.SetBlock(location, Block.Air);
+                    world.SpawnFallingBlock(location, material);
+                    return true;
+                }
+                return false;
+            });
```

Comment:
I'm not sure if this should run in its own thread running possibly minor sync code just to solve an async warning.

Reply:
I have made this sync again, this will have the async warning but as you said I don't think it's too important to resolve this with this.